types/src/main/scala/roc/types/decoders.scala

```diff
@@ -23,7 +26,7 @@ object decoders {
 
   implicit val stringElementDecoder: ElementDecoder[String] = new ElementDecoder[String] {
     def textDecoder(text: String): String         = text
-    def binaryDecoder(bytes: Array[Byte]): String = bytes.map(_.toChar).mkString
```

Comment:
Nice catch. My brain wasn't working when I originally wrote that for some reason. I would be curious as to how testing missed that though.